submissions/kutembaChinambu/database/init.sql

```diff
@@ -0,0 +1,63 @@
+<!-- This is a query that creates the food table.-->
+
+CREATE TABLE food 
+(
+id INT NOT NULL AUTO_INCREMENT,
+fast_food VARCHAR(100) NOT NULL,
```

Comment:
lines 6-9 are not needed. It should be a general type which could then be of datatype ENUM (category 1 ..... blah blah blah)